<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一组随机数</string>
  </property>
  <property name="windowIcon">
   <iconset resource="apprcc.qrc">
    <normaloff>:/pic/pic/favicon.ico</normaloff>:/pic/pic/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonConnectBluetooth">
      <property name="text">
       <string>连接蓝牙</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>数据类型</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBoxDataType">
      <item>
       <property name="text">
        <string>整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>小数</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>个数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBoxNumbers">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>最小值</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditMinValue">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>最大值</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditMaxValue">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButtonSendData">
      <property name="text">
       <string>生成数据并通过蓝牙传输到手机软件</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonSearchBluetooth">
      <property name="text">
       <string>搜索蓝牙</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxSelectBluetooth"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionJSP">
   <property name="text">
    <string>静态作业车间</string>
   </property>
   <property name="toolTip">
    <string>静态作业车间</string>
   </property>
  </action>
  <action name="actionFJSP">
   <property name="text">
    <string>静态柔性作业车间</string>
   </property>
  </action>
  <action name="actionFSP">
   <property name="text">
    <string>静态流水车间</string>
   </property>
  </action>
  <action name="actionHFSP">
   <property name="text">
    <string>静态混合流水车间</string>
   </property>
  </action>
  <action name="actionClassic">
   <property name="text">
    <string>Classic</string>
   </property>
  </action>
  <action name="actionTimetable">
   <property name="text">
    <string>Timetable</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="iconText">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>编辑</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="apprcc.qrc"/>
 </resources>
 <connections/>
</ui>
